Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -237,7 +236,7 @@ def revoke_user_session(self, user):
 
 
 def suppress_errors_with_404_code(func):
-    def wrapper(client: MsGraphClient, args: Dict):
+    def wrapper(client: MsGraphClient, args: dict):
```

Comment:
Usage of `typing` collections (`typing.Dict`, `typing.List`, `typing.Tuple`, etc.) is deprecated since Python 3.9 (https://peps.python.org/pep-0585/#implementation - Ctrl + F "deprecated")